Solutions/Torq/Playbooks/Torq-Sentinel-Incident-Trigger/azuredeploy.json

```diff
@@ -0,0 +1,165 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "metadata": {
+        "title": "Notify Sentinel Incident Creation and Update to Torq Webhook",
```

Comment:
The playbook is missing with some properties in metadata 
postdeployement
prerequisites 
please go through this playbook once to check missing metadata properties 
https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/Recorded%20Future/Playbooks/Enrichment/RecordedFuture-IOC_Enrichment/azuredeploy.json